src/librbd/api/Group.cc

```diff
@@ -1049,6 +1175,46 @@ int Group<I>::snap_list(librados::IoCtx& group_ioctx, const char *group_name,
   return 0;
 }
 
+template <typename I>
+int Group<I>::snap_rollback(librados::IoCtx& group_ioctx,
+                            const char *group_name, const char *snap_name,
+                            ProgressContext& pctx)
+{
+  CephContext *cct = (CephContext *)group_ioctx.cct();
+  librados::Rados rados(group_ioctx);
```

Comment:
nit: unused

Reply:
@trociny  It seems we could not just remove `librados::Rados rados(group_ioctx);` (the error as below) and I keep it the same. Others all fixed, please review again. Thank you.
```
/clove/swb/ceph/src/librbd/api/Group.cc: In function ‘int librbd::api::{anonymous}::group_snap_rollback_by_record(librados::IoCtx&, const cls::rbd::GroupSnapshot&, const string&, const string&, librbd::ProgressContext&)’:
/clove/swb/ceph/src/librbd/api/Group.cc:335:14: error: expected primary-expression before ‘.’ token
     r = rados.ioctx_create2(group_snap.snaps[i].pool, image_io_ctx);
              ^
```

Reply:
I meant to remove it from `snap_rollback` only, while it looks like you tried to remove it from `group_snap_rollback_by_record`?

Reply:
Ah, yeah -- my fault, sorry for that. Anyway, already removed from `snap_rollback`, please take a look. Thanks.